ui-core/src/styles/inputs/passwordInput.css

```diff
@@ -1,6 +1,6 @@
 .password-input {
     .input {
-        width: 235px !important;
+        width: 14.6875rem !important; /* Converted from 235px */
```

Comment:
3 decimals accuracy is enough

https://stackoverflow.com/questions/35492847/how-granular-are-css-rem-units